src/ubloxGPS.h

```diff
@@ -670,6 +676,8 @@ class ubloxGPS
     bool  getVersion(String& swVersion, String& hwVersion, String& extVersion);
     bool  setGNSS(uint8_t gnssMask);
     bool  setMode(ubx_dynamic_model_t dynModel);
+    bool  getMode(ubx_dynamic_model_t& dynModel);
+    bool  updateMode();
```

Comment:
Just from the method signature, it's not clear to me what `updateMode()` does.  Does it flush changes made by `setMode()` ?  Header doc comments might help

Reply:
It is a method to tell the module to retrieve the data.  The driver is setup somewhat asynchronously so you'd make a command request and then filter the response from all of the mix of data sent from the module.